lib/Runtime/Library/WasmLibrary.cpp

```diff
@@ -59,6 +59,14 @@ namespace Js
             );
         END_LEAVE_SCRIPT_INTERNAL(scriptContext)
 
+        Js::PropertyRecord const * mainProp = nullptr;
+        scriptContext->GetOrAddPropertyRecord(_u("main"), lstrlen(_u("main")), &mainProp);
+        Js::Var mainFunc = Js::JavascriptOperators::OP_GetProperty(exportObject, mainProp->GetPropertyId(), scriptContext);
+        Js::ScriptFunction* f = Js::AsmJsScriptFunction::FromVar(mainFunc);
+        Js::CallInfo info(Js::CallFlags_New, 1);
+        Js::Arguments arg0(info, &mainFunc);
+        Js::JavascriptFunction::CallFunction<true>(f, f->GetEntryPoint(), arg0);
```

Comment:
seems like this is now requiring an export function. I think we will want start function to be an optional out parameter of LoadWasmScript, and if it is not null only then we call it.